src/commands/repoRelease.js

```diff
@@ -24,44 +24,42 @@ const program = new commander.Command();
 const cliOptions = require('./cliOptions.js');
 
 program
-    .name('taoRelease createRelease')
+    .name('taoRelease ghRelease')
     .usage('[options]')
     .option(...cliOptions.debug)
+    .option(...cliOptions.releaseVersion)
     // options with defaults
     .option(...cliOptions.baseBranch)
     .option(...cliOptions.branchPrefix)
     .option(...cliOptions.origin)
     .option(...cliOptions.releaseBranch)
-    .option(...cliOptions.wwwUser)
     // options which fall back to user prompts if undefined
-    .option(...cliOptions.pathToTao)
-    .option(...cliOptions.extensionToRelease)
-    .option(...cliOptions.versionToRelease)
-    .option(...cliOptions.updateTranslations)
     .option(...cliOptions.releaseComment)
     .parse(process.argv);
 
 if (program.debug) {
     log.info(program.opts());
 }
 
-const release = require('../release.js')({ ...program.opts(), subjectType: 'extension' });
+const release = require('../release.js')({ ...program.opts(), subjectType: 'repo' });
 
-async function createRelease() {
+async function npmRelease() {
```

Comment:
Since it's internal, the function name can be updated to match the module's purpose.

Reply:
my bad :/